hie-apply-refact/hie-apply-refact.cabal

```diff
@@ -0,0 +1,30 @@
+name:                hie-apply-refact
+version:             0.1.0.0
+synopsis:            Haskell IDE Apply Refact plugin
+license:             BSD3
+license-file:        ../LICENSE
+author:              Alan Zimmerman
+maintainer:          alan.zimm@gmail.com (for now)
+copyright:           2015 TBD
+category:            Web
+build-type:          Simple
+-- extra-source-files:
+cabal-version:       >=1.10
+
+library
+  exposed-modules:     Haskell.Ide.ApplyRefactPlugin
+  build-depends:       base >= 4.7 && < 5
+                     , aeson
+                     , apply-refact
```

Comment:
I thought apply-refact was integrated into hlint, and we used it via hlint invocation, do we need that dependency? I don't see any usage of apply-refact modules.